miden-lib/asm/note_scripts/basic.masm

```diff
@@ -50,10 +50,12 @@ end
 #! 
 #! Requires that the account exposes: miden::wallets::basic::receive_asset procedure.
 #!
-#! Inputs: [target_account_id]
+#! Note Inputs: [[target_account_id, *, *, *]]
+#! Stack Inputs: []
 #! Outputs: []
```

Comment:
I'm not sure about this format for specifying note inputs. I would probably keep them separate from the stack to keep things a bit more consistent.

We do already say in the intro sentence what should be in note inputs. If we wanted to make these more explicit, I would probably have a separate section below (above the `FAILS` section) for that.